crypto/asn1/asn_mime.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include <ctype.h>
+#include "internal/ctype.h"
```

Comment:
I've changed it back.  All uses of tolower were guarded by ossl_isXXX calls, so they were safe.

You are correct, it looks more consistent and doesn't flag as a problem with ossl_tolower.